apps/prairielearn/src/lib/assessment.sql

```diff
@@ -161,6 +161,46 @@ FROM
 WHERE
   a.id = $assessment_id;
 
+-- BLOCK select_assessment_instance_max_points
```

Comment:
I'd name this one of the following so that the side effect of locking is clear:

- `select_and_lock_assessment_instance_max_points`
- `select_assessment_instance_max_points_for_update`

Reply:
Done in c4d35a160.